cpmpy/solvers/ortools.py

```diff
@@ -355,7 +355,7 @@ def transform(self, cpm_expr):
         """
         cpm_cons = toplevel_list(cpm_expr)
         supported = {"min", "max", "abs", "element", "alldifferent", "xor", "table", "negative_table", "cumulative", "circuit", "inverse", "no_overlap"}
-        cpm_cons = no_partial_functions(cpm_cons, safen_toplevel=frozenset({"div", "mod"})) # before decompose, assumes total decomposition for partial functions
+        cpm_cons = no_partial_functions(cpm_cons, safen_toplevel=frozenset({"div", "mod", "element"})) # before decompose, assumes total decomposition for partial functions
```

Comment:
right, so this would be for all elements... even the 1d ones, where we know it is often not totally formulated, but intended to be total by the user... I see